activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/swagger/QuerySwaggerITSupport.java

```diff
@@ -62,5 +68,12 @@ public void generateSwagger() throws Exception {
                 );
                 Files.write(new File("target/swagger.yaml").toPath(), new YAMLMapper().writeValueAsBytes(jsonNodeTree));
             });
+
+        ObjectMapper mapper = new ObjectMapper();
+
+        JsonNode generatedSwagger = mapper.readTree(new File("target/swagger.json"));
+        JsonNode expectedSwagger = mapper.readTree(swaggerExpectedResource.getFile());
+
+        assertThat(generatedSwagger).isEqualTo(expectedSwagger);
```

Comment:
I think it's better moving the assertions to `QuerySwaggerIT` that has already some assertions around the generated swagger and keep `QuerySwaggerITSupport ` for the generation only.